### DISCUSSION TITLE
Feature request: Timed anonymous discussions 👻

### DISCUSSION THREAD

**Discussion by levindixon** [community/community#37390] [Discussions]
👋 Hello Discussions crew!

I'd like to be able to create a discussion where responses are anonymous for a period of time before revealing who said what. The goal being to remove things like authority bias (along with _many_ other kinds of bias) that come along with a discussion response being associated with an identity. 

My hypothesis is that by removing bias from important discussions people will be more likely to contribute and that the diversity of perspectives would improve, ultimately creating higher quality outcomes ❤️ 

Admittedly the _temporary_ aspect is a bit half-baked 😅 I think it's important for people to know that their contributions to the discussion will be attributed to them once the discussion is "complete". This might also serve as a kind of timebox which I suppose could be its own feature all together 🤷 

Thanks for reading, keep up the amazing work! 😄

**Comment by ossanna16** [comment]
Hi there @levindixon and welcome to our community! Thank you for the suggestion 🙂
